test/torchaudio_unittest/common_utils/rnnt_utils.py

```diff
@@ -1,9 +1,174 @@
 import unittest
 import random
 import torch
-from torchaudio.prototype.rnnt_loss import RNNTLoss
-
-from .numpy_transducer import NumpyTransducerLoss
```

Comment:
The test code in [autograd_impl](https://github.com/pytorch/audio/blob/main/test/torchaudio_unittest/rnnt/autograd_impl.py#L15) uses ``NumpyTransducerLoss``,  we should also update that.